code/modules/mob/living/carbon/life.dm

```diff
@@ -84,12 +84,7 @@
 
 	if(breath)
 		loc.assume_air(breath)
-		//spread virus2
-		if(virus2.len > 0)
-			if(prob(10) && get_infection_chance(src))
-				for(var/mob/living/carbon/M in view(1,src))
-					src.spread_disease_to(M)
-
+		air_update_turf()
```

Comment:
This is actually unrelated to disease1--not sure if we really need this or not: https://github.com/tgstation/-tg-station/pull/16161